controllers/forms/globalactions.js

```diff
@@ -16,7 +16,7 @@ module.exports = async (req, res, next) => {
 	res.locals.actions = actionChecker(req);
 
 	//make sure they have any global actions, and that they only selected global actions
-	if (!res.locals.actions.anyGlobal || res.locals.actions.anyValid > res.locals.actions.anyGlobal) {
+	if (!res.locals.actions.anyGlobal || res.locals.actions.validActions.length > res.locals.actions.anyGlobal) {
```

Comment:
Is `res.locals.actions.anyGlobal` already a number? You're comparing the length of an array to it

Reply:
If they select 0 valid actions, then by extension, none of those actions are valid
so the amount of valid actions will not be greater than the number of global actions.
So I am checking first whether there are > 0 global actions (not all valid actions are global) because `!res.locals.actions.anyGlobal` will be `false` if it is 0.
An alternative could be to check that `validActions.length > 0` and length is equal to `anyGlobal`.

I think just the variable names kinda suck. They were from an old commit where "anyXXX" were boolean. I will instead change them to "numXXX" to make it more clear that they are numbers of that type of actions